contracts/src/v0.8/feeds/test/DualAggregator.t.sol

```diff
@@ -1968,14 +1969,14 @@ contract WithdrawFunds is ConfiguredDualAggregatorBaseTest {
     vm.startPrank(USER);
     vm.expectRevert(DualAggregator.OnlyOwnerAndBillingAdminCanCall.selector);
 
-    aggregator.withdrawFunds(USER, 42);
+    s_aggregator.withdrawFunds(USER, 42);
   }
 
   // TODO: need to run a transmit to ensure the user has a lot to withdraw
   // function test_RevertIf_InsufficientBalance() public {
   //   vm.expectRevert("insufficient balance");
   //
-  //   aggregator.withdrawFunds(USER, 1e9);
+  //   s_aggregator.withdrawFunds(USER, 1e9);
```

Comment:
Now that we have the transmits implemented, could you add this test case?

Reply:
done